service-front/psalm.xml

```diff
@@ -8,7 +8,6 @@
 >
     <projectFiles>
         <directory name="module/Application/src" />
-        <directory name="../shared/logging/module/MakeLogger/src/" />
```

Comment:
Do you not wish to carry on scanning this?  Or has it moved or something?

Reply:
psalm can find it anyway as it uses the generated autoload.php file, so this is redundant.

Reply:
Actually, though, I think you're right: one of the configurations should scan that directory. Or maybe the shared directory needs its own scan.